src/local-editor-service.ts

```diff
@@ -100,25 +102,34 @@ export class LocalEditorService extends EditorService {
     }
 
     if (location.kind === 'attributeValue') {
-      const domModules = document.getByKind('dom-module');
-      if (domModules.size > 1) return;
-      const domModule = domModules.values().next().value;
-      if (!domModule || !domModule.id) return;
+      const domModule = this.getAncestorDomModuleForElement(document, location.element);
+      if (!domModule || !domModule.id)
+        return;
       const outerElement = document.getOnlyAtId('element', domModule.id);
-      if (!outerElement) return;
+      if (!outerElement)
+        return;
       const sortPrefixes = this._createSortPrefixes(outerElement);
-      const innerElement = document.getOnlyAtId('element', location.element.nodeName)!;
-      const innerAttribute = innerElement.attributes.find((value) => value.name === location.attribute)!;
+      const innerElement =
+          document.getOnlyAtId('element', location.element.nodeName)!;
+      const innerAttribute = innerElement.attributes.find(
+          (value) => value.name === location.attribute)!;
+      const attributeValue =
+          dom5.getAttribute(location.element, innerAttribute.name)!;
+      const hasDelimeters = attributeValue.match(/^\s*(\{\{|\[\[)/);
```

Comment:
minor: when you just want a boolean, prefer `regexp.test(string)` over `string.match(regexp)`